### PR TITLE
Removed implementation open_data_schema_map_load and alter. This was …

### PR DESCRIPTION
**Pull request by jacintocapote**
Removed two hooks (open_data_schema_map_endpoints_alter, open_data_schema_map_load) because this is replaced with features integration.

### CODE REVIEW

**Comment by acouch** [comment]
Why are you removing ``open_data_schema_ckan_open_data_schema_map_endpoints_alter()`` and ``open_data_schema_ckan_open_data_schema_map_load()``?

Those provide callbacks that load the endpoints in a more efficient way than using tokens.

File: modules/open_data_schema_ckan/open_data_schema_ckan.module [hunk 1]
```diff
@@ -289,8 +218,7 @@ function open_data_schema_ckan_open_data_schema_map_results_alter(&$result, $mac
       $list = array();
       $response['help'] = t("Return a list of the names of the site's datasets (packages).\n\n :param limit: if given, the list of datasets will be broken into pages of\n at most ``limit`` datasets per page and only one page will be returned\n at a time (optional)\n :type limit: int\n :param offset: when ``limit`` is given, the offset to start returning packages from\n :type offset: int\n\n :rtype: list of strings\n\n");
       foreach ($response['result'][0] as $num => $data) {
-        // This endpoint just produces a list.
-        $list[] = $data['name'];
+        $list[] = $data;
```
Comment (dafeder) [inline modules/open_data_schema_ckan/open_data_schema_ckan.module]: Why did you change this? It changes the output for the endpoint. See api/3/action/package_list on the probo site vs http://dan.dkandemo.nuamsdev.com/api/3/action/package_list

File: modules/open_data_schema_ckan/open_data_schema_ckan.module [hunk 2]
```diff
@@ -6,77 +6,6 @@
...
-  $records[] = 'ckan_site_read';
-  $records[] = 'ckan_revision_list';
-  $records[] = 'ckan_package_revision_list';
-}
-
-/**
- * Implements hook_open_data_schema_map_load().
- */
-function open_data_schema_ckan_open_data_schema_map_load($machine_name) {
-  if ($machine_name == 'ckan_site_read') {
```
Comment (dafeder) [inline modules/open_data_schema_ckan/open_data_schema_ckan.module]: These endpoints are now missing from the list of APIs, they are not defined in open_data_schema_map_dkan's features. See admin/config/services/odsm on the probo site vs http://dan.dkandemo.nuamsdev.com/admin/config/services/odsm

**Comment by jacintocapote** [comment]
@acouch We removed because are re-declarating in both places https://github.com/NuCivic/open_data_schema_map_dkan/blob/master/open_data_schema_map_dkan.features.inc the endpoints. With schema_map_load ald endpoints_alter we are creating endpoints to ODSM without any possibility to alter the mapping,....
